neo/core/container.py

```diff
@@ -73,25 +75,132 @@ def filterdata(data, targdict=None, objects=None, **kwargs):
         results = []
         for key, value in sorted(targdict.items()):
             for obj in data:
-                if (hasattr(obj, key) and getattr(obj, key) == value and
-                        all([obj is not res for res in results])):
+                if (hasattr(obj, key) and getattr(obj, key) == value and all(
+                        [obj is not res for res in results])):
```

Comment:
I don't think this change improves readability and the line also doesn't seem >100 characters. Was this a pep8speaks suggestion?

Reply:
That was a pep8speaks suggestion. In my IDE the lines would be 115 characters long when combined